packages/account/src/Sections/Security/ApiToken/api-token.jsx

```diff
@@ -51,8 +54,10 @@ class ApiToken extends React.Component {
 
         if (!token_name) {
             errors.token_name = localize('Please enter a token name.');
-        } else if (token_name.length < 2 || token_name.length > 32) {
+        } else if (token_name.length < MIN_TOKEN) {
             errors.token_name = localize('Length of token name must be between 2 and 32 characters.');
+        } else if (token_name.length > MAX_TOKEN) {
+            errors.token_name = localize('Maximum 32 characters.');
         } else if (!/^[A-Za-z0-9\s_]+$/g.test(token_name)) {
```

Comment:
this should be the second else if instead of the last one

Reply:
@mustofa-binary  thanks, yes I think about that, but as saw it already implemented that way, I think it would be good. I just fix it